shared/js/ui/models/site.es6.js

```diff
@@ -292,7 +288,8 @@ Site.prototype = window.$.extend({},
             const pixelParams = ['epbf',
                 {category: category},
                 {siteUrl: encodeURIComponent(siteUrl)},
-                {upgradedHttps: upgradedHttps.toString()}
+                {upgradedHttps: upgradedHttps.toString()},
+                {tds: this.tds}
             ]
 
             for (let tracker in trackerObjects) {
```

Comment:
In order to get the matched surrogates on the breakage pixel in row 304 below, the reason has to be 'surrogate'. It's currently 'matched rule - surrogate'. Feel free to either update the check below to `trackerDomains[domain].reason === 'matched rule - surrogate' or update the reason to be 'surrogate'. Up to you.